core/src/test/java/org/commonjava/maven/ext/manip/impl/DistributionEnforcingManipulatorTest.java

```diff
@@ -366,7 +366,7 @@ public void before()
 
         final GalleyAPIWrapper wrapper = new GalleyAPIWrapper( galleyInfra );
 
-        manipulator = new DistributionEnforcingManipulator( wrapper );
+        manipulator = new DistributionEnforcingManipulator( );
+        FieldUtils.writeField (manipulator, "galleyWrapper", wrapper, true);
     }
```

Comment:
What's the advantage of doing it this way? Doesn't this risk something breaking in weird ways if the class itself cannot specify the minimum parameter set for construction?